k8s/test-environment/deploy-test.yaml

```diff
@@ -0,0 +1,196 @@
+        image: geekzone/backend:0.1.${CIRCLE_PREVIOUS_BUILD_NUM}
+        command: ['python3', 'manage.py', 'wait_for_database']
+        envFrom:
+          - secretRef:
+              name: testing-secrets 
+      containers:
+      - name: django-backend
+        image: geekzone/backend:0.1.${CIRCLE_PREVIOUS_BUILD_NUM}
+        envFrom:
+          - secretRef:
+              name: testing-secrets
+        resources:
+          limits:
+            memory: "100Mi"
+            cpu: 50m
+          requests:
+            memory: "50Mi"
+            cpu: 30m
+      - name: frontend-proxy
+        image: geekzone/frontend:0.1.${CIRCLE_PREVIOUS_BUILD_NUM}
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+        readinessProbe:
+          tcpSocket:
+            port: 8080
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        livenessProbe:
+          tcpSocket:
+            port: 8080
+          initialDelaySeconds: 15
+          periodSeconds: 20
+        resources:
+          limits:
+            memory: "50Mi"
+            cpu: 50m
+          requests:
+            memory: "30Mi"
+            cpu: 30m
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: test-gz-web
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}-test.geek.zone
+#    external-dns.alpha.kubernetes.io/ttl: "120" #optional
+spec:
+  type: ClusterIP
+  selector:
+    app: gz-web
+  ports:
+  - port: 80
+    targetPort: 8080
+
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgres
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+spec:
+  serviceName: postgres
+  replicas: 1
+  selector:
+    matchLabels:
+      name: postgres
+  template:
+    metadata:
+      labels:
+       name: postgres
+    spec:
+      containers:
+        - name: postgres
+          image: postgres
+          ports:
+            - containerPort: 5432
+          envFrom:
+          - secretRef:
+              name: postgres-secrets
+          volumeMounts:
+            - name: data
+              mountPath: /var/lib/postgresql
+          resources:
+            limits:
+              memory: "50Mi"
+              cpu: 50m
+            requests:
+              memory: "30Mi"
+              cpu: 30m
+      volumes:
+        - name: data
+          emptyDir: {}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+spec:
+  type: ClusterIP
+  ports:
+    - port: 5432
+  selector:
+    name: postgres
+
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: test-ingress
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+  annotations:
+    nginx.ingress.kubernetes.io/app-root: /memberships/register 
+#    nginx.ingress.kubernetes.io/default-backend: nginx-custom-default-backend 
+#    nginx.ingress.kubernetes.io/custom-http-errors: "404,415"
```

Comment:
Another set of commented code, please avoid

Reply:
I'll delete this PR and create a new one